recipes/calib/meta.yaml

```diff
@@ -17,8 +17,6 @@ build:
 requirements:
   build:
     - make
-    - zlib
```

Comment:
Zlib is used at compile time. Shouldn't we keep this?

Reply:
The host section is enough. The build section is really just for the compilers and the build-machinery ... e.g. cmake etc.

Reply:
No, it's in the host section, since it's a library that needs to be linked against.